src/binfhe/lib/rgsw-acc-cggi.cpp

```diff
@@ -75,7 +75,6 @@ RingGSWEvalKey RingGSWAccumulatorCGGI::KeyGenCGGI(const std::shared_ptr<RingGSWC
 
     DiscreteUniformGeneratorImpl<NativeVector> dug;
     NativeInteger Q{params->GetQ()};
-    dug.SetModulus(Q);
```

Comment:
would like to make sure that dug doesn't need to be constructed with the modulus "Q".

Reply:
It Doesn't, which is why I removed it. Modulus is set on line 82 when dug is first used.